requests_cache/__init__.py

```diff
@@ -18,15 +19,17 @@
         # will take approximately 5 seconds instead 50
 
 
-    :copyright: (c) 2012 by Roman Haritonov.
+    :copyright: (c) 2021 by Roman Haritonov.
```

Comment:
could also be 2012-2021, but I don't know how these things should be handled.

Reply:
Typically the copyright year is just the year it was most recently released/updated.